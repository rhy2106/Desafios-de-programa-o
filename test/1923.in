8 2
Jonathan Pedro
Rerisson Lucas
Rerisson Jonathan
Lucas Jonathan
Pedro Juan
Lucas Juan
Barbara Carla
Carla Beto