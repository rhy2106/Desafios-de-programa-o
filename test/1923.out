4
Jonathan
Juan
Lucas
Pedro
